.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+# Issue
+<!-- What issue is this PR trying to solve? -->
+
+
+# Solution
+<!-- A summary of the solution addressing the above issue -->
+
+
+# Context
+<!-- What is some specialized knowledge relevant to this project/technology -->
+
+
```

Comment:
Should there be a section for `testing`?

Reply:
A section for testing would be great.

The Juju core team has a [QA steps section](https://github.com/juju/juju/blob/2.9/PULL_REQUEST_TEMPLATE.md), which I like a lot. In addition to making the process of QA'ing a PR more straightforward, it's a great place to share little tips and tricks on testing things w/ Juju.

Reply:
added a testing section in 1204c3eb2c436b6458631445e0bb8ac9451dcaba